tests/fixtures/test_validators/validators/check_hostname.py

```diff
@@ -0,0 +1,8 @@
+"""Validate hostname is valid."""
+from pydantic import BaseModel
+
+
+class CheckHostname(BaseModel):
+    """Validate hostname is valid."""
+
+    hostname: str
```

Comment:
How's this file relate to / differ from `tests/fixtures/test_validators_pydantic/pydantic_validators/models/hostname.py`?